usermods/M5Stick2_Visualiser/README.md

```diff
@@ -0,0 +1,57 @@
+# Using the M5Stick C Plus2
+Connect the stick to 5V via your favourite method (USB-C/5V). DO NOT POWER THE LEDS FROM THE STICK OR ITS BATTERY!
+
+## Hardware
+
+***
+![Hardware](images/m5stick_c_plus2.webp)
+
+Connect the data pin of your strips to any of the GPIO pins. Connect GND to any of the GND pins.
+
+## Library used
+
+Used the eSPI TFT library directly, as adding the M5Stack library would cause too much of an overhead.
+
+[Bodmer/TFT_eSPI](https://github.com/Bodmer/TFT_eSPI)
+
+## Setup
+
+In the `platformio.ini` file, you must change the environment setup to build for just the m5stick platform by adding the following target and commeting the other ones:
+
+```
+[env:m5stick]
+board = esp32dev
+build_unflags = ${common.build_unflags}
+build_flags = ${common.build_flags_esp32} 
+  -D BTNPIN=39
+  -D LEDPIN=19
+  -D IRPIN=19
+  -D WLED_DEBUG=1
+  # Display config
+  -D ST7789_DRIVER=1
+  -D USERMOD_M5STICK2_VISUALISER=1
+  -D TFT_WIDTH=135
+  -D TFT_HEIGHT=240
+  -D TFT_MOSI=15
+  -D TFT_BL=27
+  -D TFT_SCLK=13
+  -D TFT_DC=14
+  -D TFT_RST=12
+  -D TFT_CS=5
+  -D SPI_FREQUENCY=20000000
+  -D USER_SETUP_LOADED
+  -D USERMOD_AUDIOREACTIVE -D UM_AUDIOREACTIVE_USE_NEW_FFT
```

Comment:
Hi, this is the "old way" for activating the audioreactive usermod. It's better if you use references to standard flags. Also you may want to to add flags for the onboard mic pins.
```ini
  ${esp32.AR_build_flags} ;; default flags required to properly configure ArduinoFFT
  -D I2S_SDPIN=34 -D I2S_WSPIN=0 -D I2S_CKPIN=-1 -D MCLK_PIN=-1  ;; PDM microphone pins
  -D DMTYPE=5 -D SR_SQUELCH=5 -D SR_GAIN=75    ;; SPM1423 specific sound settings
  -D UM_AUDIOREACTIVE_ENABLE ;; enable AR by default
```

The example is actually from the M5StickC, so your pins might be different.